api/models/call.go

```diff
@@ -144,6 +144,12 @@ type Call struct {
 	// Error is the reason why the call failed, it is only non-empty if
 	// status is equal to "error".
 	Error string `json:"error,omitempty" db:"error"`
+
+	// AppMetadata contains a copy of the app metadata
+	AppMetadata Metadata `json:"app_metadata,omitempty" db:"app_metadata"`
+
+	// RouteMetadata contains a copy of the app metadata
+	RouteMetadata Metadata `json:"route_metadata,omitempty" db:"route_metadata"`
```

Comment:
not sure about persisting these in the call object, this struct is something of a mess that needs sorted out. the `db:` fields are to return from the API to the user for viewing, at present this is a very small number of fields that tend to change from call to call (timestamps, stats, status) vs. configuration which is present on a route, which are easily looked up on the route (and in the somewhat near future, these will be versioned and easier to audit older versions of). the `json:` fields double to 1) show any fields in the API to the user that also have a non-`db:"-"` and 2) to store all call configuration in an mq message for async.  in any event, adding these to each call in the db bloats the bytes we store per call, each call specifies which route it's from, and metadata will be exposed on the app and route there.